client/components/banner/style.scss

```diff
@@ -261,7 +261,8 @@
 	margin-top: -8px;
 }
 
-.is-horizontal .dismissible-card__close-button, .is-horizontal .banner__close-icon {
```

Comment:
Minor nit: it may be worth reverting the CSS change to this file, as it doesn't feel right to include only a linting update in this otherwise unrelated PR.

Reply:
I've done that.